pkgs/applications/networking/browsers/ladybird/default.nix

```diff
@@ -30,6 +31,18 @@ in gcc11Stdenv.mkDerivation {
     hash = "sha256-XQj2Bohk8F6dGCAManOmmDP5b/SqEeZXZbLDYPfvi2E=";
   };
 
+  postPatch = ''
+    mkdir serenity
+    cp -r ${serenity}/* serenity
+    chmod -R +w serenity
+  ''
+  # https://github.com/SerenityOS/serenity/issues/10055
+  + lib.optionalString stdenv.isDarwin ''
+    sed -i '/MACOSX_BUNDLE/d' CMakeLists.txt
+    substituteInPlace serenity/Meta/Lagom/CMakeLists.txt \
+      --replace "@rpath" "${placeholder "out"}/lib"
+  '';
```

Comment:
Do you know how to fix this upstream? If yes, could you send them a patch?

(linking the upstream issue here to cross-reference: https://github.com/SerenityOS/serenity/issues/10055)

Reply:
I have no clue about the RPATH handling issue: replacing `@rpath` with `${placeholder "out"}/lib` is certainly not a solution to take upstream.

Reply:
Now using `install_name_tool` in `postInstall`.

Reply:
Since modifying serenity is no longer necessary, can we switch back from `srcs` to `let..in`?

Reply:
Done.

Reply:
Thanks!